vignettes/Validation.Rmd

```diff
@@ -24,23 +24,39 @@ iso3c <- "ATG"
 pop <- hypatia:::get_population(iso3c)
 
 # Scale it for speed
-pop$n <- round(pop$n/10)
+pop$n <- as.integer(pop$n / 10)
 
 # Create our equivalent parameters
+R0 <- 2
 time_period <- 200
+tt_contact_matrix <- 0
 
-parameters <-  hypatia::get_parameters(
+parameters <- hypatia:::get_parameters(
```

Comment:
triple colon is not needed here (or above on l. 24) as `get_parameters` is an exported function

```suggestion
parameters <- hypatia::get_parameters(
```